sass/typography/_headings.scss

```diff
@@ -117,6 +117,11 @@ h6 {
 	font-weight: bold;
 }
 
+.entry-content .page-links .post-page-numbers {
```

Comment:
@nielslange Would you mind giving this another look? I don't see the `post-page-numbers` class in my tests, so things are looking like this: 

![screen shot 2018-10-29 at 2 10 37 pm](https://user-images.githubusercontent.com/1202812/47670777-6c10b380-db84-11e8-8776-cc05cc0b6a0a.png)

Reply:
@nielslange This fix should appear in the `sass/site/primary/_posts-and-pages.scss` sass file. The `_headings.scss` is specifically for grouping typographically similar elements and should only address font-related styles. Fix that and the point @kjell mentioned and we can get this merged in. 👍